apps/css-workshop/src/pages/alert.astro

```diff
@@ -39,14 +40,13 @@ import IconButton_ from '../components/IconButton.astro';
     <div class='iui-alert' data-iui-status='informational'>
       <svg-info-circular class='iui-svg-icon' data-iui-icon-color='informational' aria-hidden='true'
       ></svg-info-circular>
-      <span class='iui-alert-message'
-        >Informational message.<a
-          class='iui-anchor iui-alert-link'
-          href='#'
-          data-iui-underline='true'
-          data-iui-status='informational'>Learn more</a
-        >
-      </span>
+      <!-- prettier-ignore -->
+      <span class='iui-alert-message'>
+        Informational message.<Anchor_
+          class='iui-alert-link'
+          underline='true'
+          status='informational'
+        >Learn more</Anchor_></span>
```

Comment:
This feels pretty hacky, but it's the only way I've found to fix the white space issue.

Reply:
Doesn't `prettier-ignore` ignore the whole file? Would it be better to use `prettier-ignore-start` and `prettier-ignore-end`?